src/pages/workspace/WorkspaceMembersPage.js

```diff
@@ -219,10 +219,10 @@ class WorkspaceMembersPage extends React.Component {
     }) {
         const canBeRemoved = this.props.policy.owner !== item.login && this.props.session.email !== item.login;
         return (
-            <OfflineWithFeedback onClose={() => this.dismissError(item)} pendingAction={item.pendingAction} errors={item.errors}>
+            <OfflineWithFeedback errorRowStyles={[styles.peopleRowBorderBottom]} onClose={() => this.dismissError(item)} pendingAction={item.pendingAction} errors={item.errors}>
                 <Hoverable onHoverIn={() => this.willTooltipShowForLogin(item.login, true)} onHoverOut={() => this.setState({showTooltipForLogin: ''})}>
                     <TouchableOpacity
-                        style={[styles.peopleRow, !canBeRemoved && styles.cursorDisabled]}
+                        style={[styles.peopleRow, !item.errors && styles.peopleRowBorderBottom, !canBeRemoved && styles.cursorDisabled]}
```

Comment:
Because if there are errors, then the border bottom style is also added to the error row [here](https://github.com/Expensify/App/blob/e9920b7f2d494bfbdf444480986e97458027fb33/src/pages/workspace/WorkspaceMembersPage.js#L222-L223). If we always add it here then we'd end up in a situation like this:

<img width="374" alt="Screen Shot 2022-08-31 at 5 38 40 PM" src="https://user-images.githubusercontent.com/31285285/187648891-5b54a9d4-3178-4a90-8296-9cbe13b92293.png">

With the condition that fixes the problem:

<img width="375" alt="Screen Shot 2022-08-31 at 5 39 46 PM" src="https://user-images.githubusercontent.com/31285285/187649032-6f10e4ea-a35c-4622-8ff7-8713aa190df2.png">